app/src/main/kotlin/com/hedvig/app/feature/di/KoinInitializer.kt

```diff
@@ -129,6 +129,7 @@ class KoinInitializer : Initializer<KoinApplication> {
         viewModelModule,
         whatsNewModule,
         workManagerModule,
+        changeAddressModule,
```

Comment:
These are alphabetically sorted, if you don't mind doing that too.
I typically highlight everything, do (shift + shift) which brings up a window and type "sort lines" in there. Then only pressing enter does it for you on the highlighted text.